prow/cluster/jobs/istio/test-infra/istio.test-infra.trusted.master.yaml

```diff
@@ -29,6 +29,58 @@ postsubmits:
         secret:
           secretName: prow-deployer-service-account
 
+  - name: post-test-infra-update-boskos-config
+    cluster: test-infra-trusted
+    run_if_changed: '^boskos/resources.yaml$'
+    decorate: true
+    branches:
+      - master
+    annotations:
+      testgrid-create-test-group: "false"
+    spec:
+      containers:
+        - image: gcr.io/k8s-testimages/gcloud-in-go:v20190125-cc5d6ecff3
+          command:
+            - bash
+          args:
+            - -c
+            - |
+              gcloud auth activate-service-account --key-file=/creds/service-account.json && \
```

Comment:
Consider trying to make the command just `make -C boskos boskos-config` (aka skip bash) and set the `env: GOOGLE_APPLICATION_CREDENTIALS=/creds/service-account.json`, which may or may not work

Reply:
Okay, Let me give that a shot.

Reply:
It did not seem to work. Maybe it has to do with the fact `make` runs in subshells ... not sure. I would like to get this simplified but might need to do this in a follow-up.

Reply:
https://stackoverflow.com/a/52387709/4279440